actions.go

```diff
@@ -57,6 +57,9 @@ func PerformScan(addrStr, scriptName string, dryRun bool, cfg *Config, cfgDir st
 	if changed := getProcessors(result, baseObj, client, dryRun); changed {
 		changesDetected = true
 	}
+	if changed := getDisks(result, baseObj, client, dryRun); changed {
```

Comment:
Readability, mostly. Consider this (or better yet - try to read it as plain English):
`if changed ...`
vs. this:
`if getDisks ...`
In the latter case, it's not quite obvious what is happening here, so naming the returned value explicitly IMHO improves that, with very little cost (variables declared here are visible only in their `if` blocks).